### DISCUSSION TITLE
Laravel to Supabase is a bit slow.

### DISCUSSION THREAD

**Discussion by kennsecusana** [supabase/supabase#28728] [Questions]
Good day,

I hope this message finds you well. I am reaching out to seek your assistance with a performance issue related to my Laravel and Supabase connection. I have noticed that both in the development environment and in production, the response times are somewhat slow. Despite indexing the columns on the relevant table, even retrieving just two pieces of data without any conditions seems to be slow.

Could there be any additional configuration or settings that I might have overlooked? Your guidance would be greatly appreciated.

Thank you in advance for your help.

Best regards,
Kenn

**Comment by GaryAustin1** [comment]
You should probably provide more context on what slow is.  How you are accessing this (supavisor, direct connect, I assume not REST).  Table size, query.

**Reply by kennsecusana** [reply]
Im using DB facade and I only tested a single data to be fetched in local and production I have set the connection in the environment var using the full admin access but its slow. what are your suggestions? I tried to query only one data but its still slow I added database url and the password

**Comment by kubeden** [comment]
++same; looking for more information

**Comment by Achour** [comment]
I am running into the same problem, any solutons!!

**Reply by kubeden** [reply]
Hey. I found my solution.

What did the job for me was to:

1/ add postgres persistence inside 'database.php'

'''
"option" => [
  PDO::ATTR_PERSISTENT => "true"
]

2/ get rid of my Livewire n+1s

**Reply by Achour** [reply]
Still slow for me :x using filament as a dashboard

**Reply by Timadey** [reply]
I am experiencing the same issue as @Achour, did you find a solution to this? I am also using Filament as dashboard with Supabase PostgreSQL as db connection
